models/game.rb

```diff
@@ -89,9 +85,19 @@ def self.home_games(user, **opts)
 
     kwargs[:user_id] = user.id if user
     kwargs[:title] = opts['title'] != '' ? opts['title'] : nil
+    kwargs[:status] = %w[new active]
+    kwargs[:limit] = 1000
     fetch(user ? LOGGED_IN_QUERY : LOGGED_OUT_QUERY, **kwargs,).all.sort_by(&:id).reverse
   end
 
+  def self.profile_games(user)
+    kwargs = {}
+    kwargs[:user_id] = user.id if user
```

Comment:
why wouldn't there be a user?

Reply:
fetch(user_query, user_id: user.id)... we don't need the kwargs building

Reply:
copy and paste

Reply:
Fixed